blog/2020/issues.md

```diff
@@ -0,0 +1,39 @@
+# Github Issues Procedure
+
+Hi VisiDaters!
+
+Over the past few years our approach and relationship to Github Issues has evolved.
+Our current approach is working reasonably for us, but it might be confusing to some people at first.
+So we wanted to write up a blog post to clearly outline what our current process is.
+
+GitHub issues are a big source of tension in Open Source.
+A healthy stream of bug reports, feature requests, and conversation are the bread and butter for an open source community.
+We adore them.
+They help communicate to us how VisiData is used, and how it can be made even awesomer.
+Hearing about users' unique workflows is a definite highlight for us.
+Getting a really detailed issue with sample data and an easy repro can make it so frictionless that it hardly feels like work to fix or implement it.
+
+The dark side of issues is that there are many users with many desires, and only two committed contributors.
+Saying No on an issue can sometimes be quite a fraught thing, sometimes escalating into debates. over our No. 
+We are hoping that this process will help make the GitHub issues more of a place of opportunity, instead of a source of dread for the two of us.
+
+## The Process
+
+[\@saulpw](https://github.com/saulpw) decides what goes into the VisiData core.
+He has a strong preference that issues are only left open if they are part of the current ROADMAP.
```

Comment:
Let's use "Roadmap" to differentiate the conceptual roadmap from the `ROADMAP` file (not all open issues will be explicitly in the file).